apiary.apib

```diff
@@ -3,128 +3,156 @@ HOST: http://lore.mit.edu/
+|`{resource_id}`  |The `resource_id` sets the unique identifier (name or numerical) for a specific item to reference. |
 
-*Example: `http://lore.mit.edu/api/v1/repos` will return a json representation of all repositories.*
+*Example: `http://lore.mit.edu/api/v1/repositories/` will return a JSON representation of all repositories.*
 
-*Example: `http://lore.mit.edu/api/v1/repos/physics1` will return a json representation of the repository with name "physics1".*
+*Example: `http://lore.mit.edu/api/v1/repositories/physics1/` will return a JSON representation of the repository with name "physics1".*
 
-*Example: `http://lore.mit.edu/api/v1/repos/physics1/learningresources/1007` will return a json representation
-of the learning resource with ID= 1007.*
+*Example: `http://lore.mit.edu/api/v1/repositories/physics1/learningresources/1007/` will return a JSON representation
+of the learning resource with ID=1007.*
 
 
 # Operations
-LORE API uses the following HTTP verbs
+LORE API uses the following HTTP verbs:
 
 | Verb   | Description |
 | ---    | --- |
-|`GET`   |Select one or more items. Success returns 200 status code with json result item(s) in body.|
-|`POST`  |Create a new item. Success returns 201 status code with newly created json item in body.|
-|`PUT`   |Update an item. Success returns 200 status code with updated json item in body.|
+|`GET`   |Select one or more items. Success returns 200 status code with JSON result item(s) in body.|
+|`POST`  |Create a new item. Success returns 201 status code with newly created JSON item in body.|
+|`PUT`   |Update an item. Success returns 200 status code with updated JSON item in body.|
 |`DELETE`|Delete an item. Success returns 204 status code with no body.|
 
 # Status Codes
-The API responds with one of the following status codes.
+The API responds with one of the following status codes:
 
 | Code| Description |
 | --- | --- |
-|`200`| Success. Request completed. |
-|`201`| Success. New resource created. |
-|`204`| Success. No content to return. Only the status code returns. |
-|`400`| Bad Request - The request could not be parsed. |
+|`200`| Success - request completed. |
+|`201`| Success - new resource created. |
+|`204`| Success - no content to return, resource was successfully deleted. |
+|`400`| Bad Request - the request could not be parsed. |
 |`401`| Unauthorized - user is not logged in, cannot access resource. |
+|`403`| Forbidden - user does not have permission to access resource. |
 |`404`| Not Found - resource doesn't exist. |
-|`422`| Unprocessable Entity - The request could not be parsed due to validation errors. |
-|`500`| Server error. |
+|`422`| Unprocessable Entity - the request could not be parsed due to validation errors. |
+|`500`| Server error - the request could not be processed due to a server error. |
 
 
 # Authentication
 
-tbd.
+The REST API uses basic authentication to authenticate users using the same credentials as the main application.
```

Comment:
Yes: https://en.wikipedia.org/wiki/Basic_access_authentication

I should probably capitalize it